stacks/nifi-kafka-druid-superset-s3/druid.yaml

```diff
@@ -62,7 +62,7 @@ spec:
               min: "1"
               max: "4" # Need enough CPU cores to run multiple ingestion jobs at once
             memory:
-              limit: 4Gi
+              limit: 16Gi
```

Comment:
We also need to update https://github.com/stackabletech/demos/blob/6382cc0ed156f01dfbd18e56556af638048b50ec/stacks/stacks-v2.yaml#L221 and the same for the two demos using this stack

Reply:
Done with https://github.com/stackabletech/demos/pull/57/commits/67a29395cf6aa60771fb5ec25195513c5cb42492